### PR TITLE
added: allow specifying solver params for L2-projection

### PR DESCRIPTION
**Pull request by akva2**

### CODE REVIEW

**Comment by akva2** [comment]
jenkins build this with downstreams please

File: src/SIM/SIMinput.C [hunk 1]
```diff
@@ -807,9 +807,15 @@ bool SIMinput::parse (const TiXmlElement* elem)
...
-      GlbL2::SolverParams = myGl2Params;
+      const TiXmlElement* l2 = elem->FirstChildElement("l2params");
+      if (l2) {
+        myGl2Params = new LinSolParams;
+        myGl2Params->read(l2);
+      } else {
+        // For now use same solver parameters in the L2-projection
+        myGl2Params = new LinSolParams(*mySolParams,LinAlg::SYMMETRIC);
+      }
+        GlbL2::SolverParams = myGl2Params;
```
Comment (kmokstad) [inline src/SIM/SIMinput.C:818]: Check indent (tab/space?)

Comment (akva2) [reply src/SIM/SIMinput.C]: fixed.

**Comment by akva2** [comment]
jenkins build this with downstreams please

**Pull request merged**